docs/download.md

```diff
@@ -100,6 +100,10 @@ The list of all supported operations is provided [here](supported_ops.md).
 For a detailed list of changes, please refer to the
 [CHANGELOG](https://github.com/NVIDIA/spark-rapids/blob/main/CHANGELOG.md). 
 
+**_Note:_** Using the CUDA 450.80.02 driver in combination with the CUDA 10.1 or 10.2 toolkit may
```

Comment:
we should probabaly say 450.80.02 or newer driver. I don't think they put out a fixed version yet, once they do we would want to update and say 450.80.02 thru XXXX version

Reply:
Found that the subsequent 450.102.04 did not have the fix, but versions after that did.  Updated the docs to reflect that.

Reply:
what about 460.X drivers?  I assume same bug applies to them until certain version, if they put out fix?